### ISSUE TITLE
postgresql entry missing from wpn-xm.ini "[paths]" section.

### ISSUE THREAD

**Issue by klonos** [WPN-XM/WPN-XM#354]
```postgresql=./bin/pgsql/bin``` needs to be added.

This was mentioned in https://github.com/WPN-XM/WPN-XM/issues/300#issuecomment-67720816

This fixes #353 by the way!

**Comment by klonos** [comment]
Adding the entry makes it so that the green "Start" button for all daemons in the SCP does not crash it, but the "Start All"/"Stop All" options from the tray icon do not have any effect for the PostgreSQL entry...

1. Start the SCP.
2. Hit the green "Start" button -> all daemons start fine (including PostgreSQL).
3. Hit the red "Stop" button -> all daemons stop fine (including PostgreSQL).
4. Right click on the tray icon and select "Start All" -> all daemons start *except* the PostgreSQL one!?!
5. Right click on the tray icon and select the "Start" option for the (not started) PostgreSQL daemon -> PostgreSQL daemon starts.
6. Right click on the tray icon and select "Stop All" -> all daemons stop *except* the PostgreSQL one!?!
7. Right click on the tray icon and select the "Stop" option for the (still running) PostgreSQL daemon -> PostgreSQL daemon stops.

**Comment by jakoch** [comment]
Welcome back, Gregory.
Thank you so much for testing and reporting all the things.

Ok: postgresql path is missing ini.

**Comment by jakoch** [comment]
I thought i fixed this in #300.
And the code is there:
https://github.com/WPN-XM/server-control-panel/blob/master/src/mainwindow.cpp#L1000

**Comment by klonos** [comment]
Well, mine was a vanilla installation of the full x64 installer on Win8.1. I made sure that the old installation folder was deleted after uninstallation and before installing afresh. Don't know why the entry was missing.

The order of the entries varies a bit in my setup (the postgres line was added manually by me):

```
[paths]
logs=./logs
php=./bin/php
mongodb=./bin/mongodb/bin
memcached=./bin/memcached
mariadb=./bin/mariadb/bin
nginx=./bin/nginx
postgresql=./bin/pgsql/bin
```

...as you can see the nginx line is listed last while in your .ini is listed after the php line. Don't know if that helps at all.

**Comment by jakoch** [comment]
I figured it out. 
The config folder contains a wpnxm.ini with old data, see https://github.com/WPN-XM/WPN-XM/blob/master/configs/wpn-xm.ini
And that's the one being installed.

So the workaround is simply to delete the file, then it's re-created with new settings.

**Comment by jakoch** [comment]
And ... that seems to be the reason for the logs not opening error #356, too.
Because "notepad.exe" is also missing, to the button action is triggered, but the editor value is empty.

**Comment by klonos** [comment]
Confirming fixed in 0.8.2. Thanx!
